app/src/main/java/com/asfoundation/wallet/ui/OneStepPaymentReceiver.java

```diff
@@ -93,13 +82,6 @@ private void startOneStepTransfer(TransactionBuilder transaction, boolean isBds)
     walletCreationText = null;
   }
 
-  private Single<Wallet> createWallet() {
-    showLoadingAnimation();
-    return paymentReceiverInteract.createWallet()
-        .observeOn(AndroidSchedulers.mainThread())
-        .doAfterTerminate(this::endAnimation);
-  }
-
   private void endAnimation() {
```

Comment:
method not used and can be removed

Reply:
There was some missing logic missing to use the animation when a wallet is being created